stl/inc/functional

```diff
@@ -2795,6 +2795,8 @@ namespace _DEPRECATE_TR1_NAMESPACE tr1 {
 } // namespace tr1
 #endif // _HAS_TR1_NAMESPACE
 
+#undef _USE_FUNCTION_INT_0_SFINAE

```

Comment:
Why don't we push the definition of `_USE_FUNCTION_INT_0_SFINAE` up into `<yvals_core.h>` and rename to `_USE_INT_0_SFINAE` instead of defining a differently-named but identical macro to control the same process in `<memory>`?